live-examples/js-examples/date-getutcmilliseconds.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js">var date1 = new Date('2018-01-24T12:38:29.069Z');
+
+console.log(date1.getUTCMilliseconds());
+// expected output: 69
+
+var date2 = new Date('2018-01-24T12:38:29.001Z');
+
+console.log(date2.getUTCMilliseconds());
+// expected output: 1
+</code>
+</pre>
```

Comment:
Please end files with a return (this comment applies to all new files in this PR).